package.json

```diff
@@ -78,7 +78,7 @@
     "sinon": "^9.0.2"
   },
   "engines": {
-    "node": ">=10.13.0"
```

Comment:
Noticed that the minimum required Node.js version is bumped to 11.14.0.

Reply:
Node.js 11 is end of life. If possible, the minimum required Node.js version should be bumped to 12.x

Reply:
https://github.com/hexojs/warehouse/issues/83#issuecomment-629762394
This can be thought of as Hexo is not ready for a PR that depends on node @ 12 and above.
I'm wondering if this PR merge should be pending so much, as there will be a lot of commits while waiting for a major Hexo update.

Reply:
Is it possible to make this PR compatible with Node.js 12?

Reply:
Yes. We need to add one more nest of then().